README.md

```diff
@@ -94,7 +111,17 @@ We can't pull the images down from ECR for an M1 Mac because they are not in ARM
 1. In the [bichard7-next-infrastructure-docker-images](https://github.com/ministryofjustice/bichard7-next-infrastructure-docker-images/) repository, run `make build-local` to just build the required images
 1. Follow the instructions in the [bichard7-next](https://github.com/ministryofjustice/bichard7-next/#building-liberty-on-arm) repository to build the Bichard Open Liberty image
 1. In the [bichard7-next-user-service](https://github.com/ministryofjustice/bichard7-next-user-service/) repository run `make build`
-1. In the [bichard7-next-ui](https://github.com/ministryofjustice/bichard7-next-ui/) repository, run `make build`
+1. In the [bichard7-next-ui](https://github.com/ministryofjustice/bichard7-next-core/tree/main/packages/ui) package, run `make build` or the top level of [bichard7-next-core](https://github.com/ministryofjustice/bichard7-next-core) run this script `./scripts/build-ui-docker.sh`
+
+## Running Packages locally
+
+1. From the root directory `npm ci`
```

Comment:
Not sure if it's worth mentioning but I always use `npm i` and never had any problems.

Reply:
How about `npm i --workspaces --include-workspace-root`?

Reply:
Seems fine? I deleted my `node_modules` and then ran `npm i --workspaces --include-workspace-root`. I was able to run `npm run test:unit -w packages/core`. Not sure the best way to test...